web-components/src/utils/schemaURL.ts

```diff
@@ -1,12 +1,11 @@
+// TODO: type UrlType and getURLInitialValue
+// this has been temporarily duplicated due to moving the RDF types to the web-registry app
+
 export interface UrlType {
   '@type': 'schema:URL';
   '@value'?: string | null;
 }
```

Comment:
I believe we could even just merge `UrlType` and `URL` into one:

```
export interface UrlType {
  '@type': 'http://schema.org/URL' | 'schema:URL';
  '@value'?: string | null;
}
```

Reply:
In the end I preferred not to refactor any of the code itself. I have it on the radar for future refactorings..